uint8  motor_id
uint16 position
